src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/CQLSessionCache.java

```diff
@@ -51,14 +51,17 @@ public class CQLSessionCache {
   public static final String ASTRA = "astra";
   /** Database type OSS cassandra */
   public static final String CASSANDRA = "cassandra";
+  /** Persistence type SSTable Writer */
+  public static final String OFFLINE_WRITER = "offline_writer";
 
   @ConfigProperty(name = "quarkus.application.name")
-  private String APPLICATION_NAME;
+  String APPLICATION_NAME;
 
   @Inject
   public CQLSessionCache(OperationsConfig operationsConfig, MeterRegistry meterRegistry) {
+    LOGGER.info("Initializing CQLSessionCache");
```

Comment:
Is this generally useful? Or only during development -- has very little information.

Reply:
This is generally useful to have a log at the beginning and at the end of this constructor which happens only once during the start up/when the first request is received.